### PR TITLE
#57 Fix can't change profile picture

### PR DESCRIPTION
**Pull request by pbson**
fix: https://github.com/OpenPaaS-Suite/esn-frontend-account/issues/57

### CODE REVIEW

**Review by piavgh — DISMISSED** [review]

File: src/linagora.esn.profile/app/show/details/profile-show-details.controller.js [hunk 1]
```diff
@@ -5,12 +5,13 @@ require('../../common/profile-api.service.js');
...
   .controller('profileShowDetailsController', profileShowDetailsController);
 
-function profileShowDetailsController(profileHelpersService) {
+function profileShowDetailsController(profileHelpersService, $scope) {
   var self = this;
 
   self.$onInit = $onInit;
 
   function $onInit() {
+    $scope.user = self.user;
```
Comment (huy-ta) [inline src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: Mh... why do we need this? Also you'll want to add some unit tests to make sure this doesn't break again.

**Review by rezk2ll — APPROVED** [review]

File: src/linagora.esn.profile/app/show/details/profile-show-details.controller.js (hunk 1 above)
Comment (huy-ta) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: By the way why is it "Partially fix" :trollface:

Comment (pbson) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: image
The reason for the bug is the missing $scope.user ( which is `undefined` in the picture above )

Comment (pbson) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: :trollface: Oops, fixed the partially part

Comment (huy-ta) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: Write the unit tests then :trollface:

File: src/linagora.esn.profile/app/show/details/profile-show-details.controller.js [hunk 2]
```diff
@@ -5,12 +5,13 @@ require('../../common/profile-api.service.js');
 angular.module('linagora.esn.profile')
   .controller('profileShowDetailsController', profileShowDetailsController);
 
-function profileShowDetailsController(profileHelpersService) {
+function profileShowDetailsController(profileHelpersService, $scope) {
```
Comment (tuanlc) [inline src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:8]:
```suggestion
function profileShowDetailsController($scope, profileHelpersService) {
```

File: src/linagora.esn.profile/app/show/details/profile-show-details.controller.js (hunk 1 above)
Comment (tuanlc) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: Question: If there is a `self.user`, why don't use `ctrl.user` in template but use $scope?

**Review by tuanlc — CHANGES_REQUESTED** [review]

File: src/linagora.esn.profile/app/show/details/profile-show-details.controller.js (hunk 1 above)
Comment (pbson) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js:14]: `$scope.user` is used multiple places in`avatar.js`, and this is where the bug coming from. Using `$scope.user` is the way it used to be done before the bug appeared also, whoever changed the code forgot to pass it in.

File: src/linagora.esn.profile/app/show/details/profile-show-details.controller.js (hunk 2 above)
Comment (pbson) [reply src/linagora.esn.profile/app/show/details/profile-show-details.controller.js]: Fixed

**Review by tuanlc — APPROVED** [review]

**Review by huy-ta — APPROVED** [review]

**Pull request merged**